backend/src/main/java/tqs/medex/pojo/SupplierPOJO.java

```diff
@@ -0,0 +1,28 @@
+package tqs.medex.pojo;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.DecimalMax;
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class SupplierPOJO {
+
+  @NotBlank
+  @Size(min = 4, max = 50)
+  private String name;
+
+  @DecimalMin(value = "-200.00", message = "Latitude Cannot be less than -200.0")
```

Comment:
Latitude and Longitude values are [-90,90] and [-180,180] respectively. Although not very important, might want to be consistent.

Reply:
My mistake I will now do the appropriate C

Reply:
I added now the Changes, check the new push